src/main.rs

```diff
@@ -51,8 +51,8 @@ const AGENT: &str = concat!(
     env!("CARGO_PKG_VERSION"),
     "\n"
 );
-/// Number of metadata GETs to do concurrently.
-const METADATA_FETCH_CONCURRENCY: usize = 6;
+/// Number of metadata GETs to do in parallel.
+const PARALLEL_METADATA_FETCHES: usize = 6;
```

Comment:
This is a bit "finger in the air". Might be better higher for larger registries. I'm not 100% on the best balance of memory use vs performance. I may bump it to `16`...

Reply:
Might be worth exposing this via GitlabConfig but I'll merge as-is for now. Thanks for the contribution :)